client/blocks/login/login-header.tsx

```diff
@@ -79,6 +78,16 @@ export function getHeaderText(
 			clientName = 'Blaze Pro';
 		} else if ( isA4AOAuth2Client( oauth2Client ) ) {
 			clientName = 'Automattic for Agencies';
+		} else if ( isWCCOM ) {
+			if ( twoFactorEnabled ) {
+				headerText = translate( 'Authenticate your login' );
+			} else if ( currentQuery.lostpassword_flow ) {
+				headerText = translate( 'Log in to your account' );
+			} else if ( showContinueAsUser ) {
+				headerText = translate( 'Log in to your account' );
+			}
+
+			headerText = translate( 'Log in to Woo with WordPress.com' );
```

Comment:
I think for the default case, it might be better to set the `clientName` here as `Woo` and not set the `headerText`. But NIT - I'm sure this will evolve more as we continue with the unification work.